soccer/planning/planner/InterceptPlanner.cpp

```diff
@@ -0,0 +1,88 @@
+#include "InterceptPlanner.hpp"
+
```

Comment:
This is almost a direct port from the original intercept planner. There should be no major functional changes.